changelogs/unreleased/7519-namespace-metadata-restore.yaml

```diff
@@ -0,0 +1,7 @@
+kind: bug
```

Comment:
This is trivial but conventionally the changelog is a one-liner, otherwise the format of the combined changelogs may be inconsistent.

Reply:
Accept. Will change this to a single line.